tests/e2e/forgejo/10-assert.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: kuttl.dev/v1beta1
 kind: TestAssert
-timeout: 180
+timeout: 1800
```

Comment:
wouldn't this make tests non fail-able for 30 min? Is this acceptable in local env when trying to test e2e?

Reply:
ah ok, haven't realised that.